public/pages/Snapshots/components/IndexSettingsInput/IndexSettingsInput.tsx

```diff
@@ -43,8 +43,15 @@ const IndexSettingsInput = ({ getIndexSettings, ignore }: IndexSettingsInputProp
           </EuiLink>
         </p>
       </EuiText>
-      <EuiFormRow>
-        <EuiTextArea value={indexSettings} onChange={onSettingsChange} placeholder={placeholderText} />
+    </>
+  );
+
+  return (
+    <>
+      <EuiSpacer size="m" />
+
+      <EuiFormRow isInvalid={showError} error={inputError} label={indexSettingsLabel}>
```

Comment:
Not blocking for this PR, but I think we should use `inputError` as the condition for `isInvalid`, that way we can avoid having two separate fields for same purpose. Does that make sense?

Reply:
isInvalid is stock Elastic ui prop for this component.

Reply:
it is used in form validation to let it know when to add error styling.